src/components/shared/SearchInput.css

```diff
@@ -119,3 +119,15 @@
 .search-field.big .close-btn {
   margin-inline-end: 8px;
 }
+
+.search-field .search-nav-buttons .active {
+  background: #00000030;
```

Comment:
Where did these colors come from?  You should be able to use the same colors as the other search field.

Reply:
Hi @darkwing !
From where Can I get the list of css variables ?
I'm unable to find this in repository.